meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr/imx8mqevk/boot.cmd

```diff
@@ -0,0 +1,7 @@
+setenv devnum ${emmc_dev}
+setenv devtype mmc
+
+fatload ${devtype} ${devnum}:1 ${fdt_addr} ${fdt_file}
+setenv bootargs console=${console} root=/dev/mmcblk${devnum}p2 rootwait rw
+fatload ${devtype} ${devnum}:1 ${loadaddr} ${image}
+booti ${loadaddr} - ${fdt_addr}
```

Comment:
boot.cmd is usually only used for loading uEnv.txt, but here you are booting directly to the kernel.

example from imx8mmevk:

```
$ cat ./meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr/imx8mmevk/boot.cmd
fatload mmc ${emmc_dev}:1 ${loadaddr} /uEnv.txt
env import -t ${loadaddr} ${filesize}
run bootcmd
```

Reply:
Nice catch.  Fixing.